spec/cli_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+module LibyearBundler
+  RSpec.describe CLI do
+    %w[--all --releases --libyears --versions --grand-total].each do |flag|
+      it "successfully runs with #{flag}" do
+        expect { described_class.new([flag]).run }
+          .not_to raise_error
```

Comment:
I'd like to have end-to-end tests, but I don't want the test suite to make network requests.